From 93a66d0a7984cc534a4df8c8ad19dd3d562a08b6 Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Wed, 17 Mar 2021 07:30:14 +0200
Subject: [PATCH] [SW-0] pll: timeout: bug fix for overlap

This patch fixes bug for timeout function. Default
value of overlap was not set to false causing it
to check wrong condition for timeout.

Change-Id: I9cdecd85d1a33a5b5f3298bb4e1c6d9a29bdd805
(cherry picked from commit 88623a798ad4c)
---

diff --git a/common/pll.c b/common/pll.c
index a3ad28b..aae91ed 100644
--- a/common/pll.c
+++ b/common/pll.c
@@ -66,7 +66,7 @@
 bool timeout_expired(u64 start_tv, u32 delay)
 {
 	u64 current, timeout_tv = start_tv + delay;
-	bool overlap;
+	bool overlap = false;
 
 	if (timeout_tv < start_tv)
 		overlap = true;
